### ISSUE TITLE
fix: button hover states

### ISSUE THREAD

**Issue by corinneling** [sparkbox/apprentice-at#293]
update button hover states to use another color so it meets color contrast guidelines

Update the buttons and navigation links to have a different styling change on hover since the color used for the border, #CEF0EE, does not have sufficient color contrast. A solution could be to change the color of the border and add a different background color on hover

The styles for the button are located in the [_button.scss file](https://github.com/sparkbox/apprentice-at/blob/master/source/scss/modules/_button.scss)

**Comment by changangus** [comment]
Hi corinneling 

Could I be assigned to this issue?

**Comment by corinneling** [comment]
yep!

**Comment by changangus** [comment]
Awesome, thanks!

**Comment by bryanbraun** [comment]
I just merged this at https://github.com/sparkbox/apprentice-at/pull/305. Thanks for the PR! 👍
